File: Clase_03/Clase_03/Numero.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Clase_03
{
    public class Numero : Comparable
    {
        // Atributos

        private int valor;

        // Constructor

        public Numero(int valor) 
        {
            this.valor = valor;
        }

        // Getter

        public int getValor()
        {
            return valor;
        }

        // Métodos de la interface
        public bool sosIgual(Comparable comparable)
        {
            return getValor() == ((Numero)comparable).getValor();
        }

        public bool sosMayor(Comparable comparable)
        {
            return (getValor() > ((Numero)comparable).getValor());
        }

        public bool sosMenor(Comparable comparable)
        {
            return ((getValor() < ((Numero)comparable).getValor()));
        }

        public override string ToString()
        {
            return getValor().ToString();
        }
    }
}

File: clase1/src/main/java/Numero.java
import comparable.Comparable;

public class Numero implements Comparable{
    private int valor;

    public Numero(int valor){
        this.valor = valor;
    }

    public int getValor(){
        return valor;
    }
    @Override
    public boolean sosIgual(Comparable otro) {
        return this.getValor() == ((Numero)otro).getValor();
    }

    @Override
    public boolean sosMenor(Comparable otro) {
        return this.getValor() < ((Numero)otro).getValor();
    }

    @Override
    public boolean sosMayor(Comparable otro) {
        return this.getValor() > ((Numero)otro).getValor();
    }
}
